db/schema.rb

```diff
@@ -65,7 +65,7 @@
   end
 
   create_table "bulk_email_jobs", force: :cascade do |t|
-    t.integer  "facility_id",     limit: 4,     null: false
+    t.integer  "facility_id",     limit: 4
```

Comment:
I'm inclined to leave it to keep the door open for cross-facility (or no facility?) bulk email support in the future.